spec/models/piece_spec.rb

```diff
@@ -1,4 +1,100 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
+  describe '#obstructed?' do
+    before do
+      @player1 = FactoryGirl.create(:player)
+      @player2 = FactoryGirl.create(:player)
+      @game = Game.create(name: 'my game', player_1: @player1, player_2: @player2)
+    end
```

Comment:
instead of instance variables. Use `let` or `let!`

http://betterspecs.org/#let